pulsar-broker/src/test/java/org/apache/pulsar/broker/web/WebServiceTest.java

```diff
@@ -354,6 +359,73 @@ public void testBrokerReady() throws Exception {
         assertEquals(res.getResponseBody(), "ok");
     }
 
+    @Test
+    public void testCompressOutputMetricsInPrometheus() throws Exception {
+
+        setupEnv(true, false, false, false, -1, false);
+
+        String metricsUrl = pulsar.getWebServiceAddress() + "/metrics/";
+
+        String[] command = {"curl", "-H", "Accept-Encoding: gzip", metricsUrl};
```

Comment:
This seems to be a new way to do tests, but I guess it's fine since curl is everywhere?